### PR TITLE
indexers-can-access-services

### PR DESCRIPTION
**Pull request by jmason-ebi**
Allow the indexers to access the service components.
Remove the ability for the indexes to read from one server and write to another.
Align core names.

### CODE REVIEW

File: indexers/src/main/java/org/mousephenotype/cda/indexers/AnatomyIndexer.java [hunk 1]
```diff
@@ -284,15 +276,15 @@ public RunStatus run() throws IndexerException, SQLException, IOException, SolrS
...
 //                }
 
                 documentCount++;
-                anatomyIndexing.addBean(anatomyTerm, 60000);
+                anatomyCore.addBean(anatomyTerm, 60000);
 
                 if (documentCount % 100 == 0){
-                    anatomyIndexing.commit();
+                    anatomyCore.commit();
                 }
```
Comment (IlincaTudose) [inline indexers/src/main/java/org/mousephenotype/cda/indexers/AnatomyIndexer.java]: Do you reckon we need both timed and forced commits?

**Comment by IlincaTudose** [comment]
Nice stats +487 −1,939

File: data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/EmapService.java [hunk 2]
```diff
@@ -1,133 +0,0 @@
...
-import org.apache.solr.client.solrj.SolrQuery;
-import org.apache.solr.client.solrj.SolrServerException;
-import org.apache.solr.client.solrj.impl.HttpSolrClient;
-import org.apache.solr.client.solrj.response.FacetField;
-import org.apache.solr.client.solrj.response.FacetField.Count;
-import org.apache.solr.client.solrj.response.QueryResponse;
-import org.apache.solr.common.SolrDocument;
-import org.apache.solr.common.SolrDocumentList;
-import org.mousephenotype.cda.solr.service.dto.BasicBean;
-import org.mousephenotype.cda.solr.service.dto.EmapDTO;
```
Comment (IlincaTudose) [inline data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/EmapService.java]: Deleted EmapDTO?

File: data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/dto/MaDTO.java [hunk 3]
```diff
@@ -18,7 +18,6 @@
 import org.apache.solr.client.solrj.beans.Field;
```
Comment (IlincaTudose) [inline data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/dto/MaDTO.java]: I would think we don't need MaDTO at all. Delete?

File: indexers/src/main/java/org/mousephenotype/cda/indexers/AnatomyIndexer.java (hunk 1 above)
Comment (jmason-ebi) [inline indexers/src/main/java/org/mousephenotype/cda/indexers/AnatomyIndexer.java]: We do not.  I'll remove the forced commits.

File: data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/EmapService.java (hunk 2 above)
Comment (jmason-ebi) [inline data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/EmapService.java]: Seems safe to delete.  Will do so.

File: data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/dto/MaDTO.java (hunk 3 above)
Comment (jmason-ebi) [inline data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/dto/MaDTO.java]: Seems safe to delete.  Will do as well.

**Comment by jmason-ebi** [comment]
Requested changes made.

**Pull request merged**